renderer/test/js_ethereum_provider_browsertest.cc

```diff
@@ -253,76 +273,33 @@ IN_PROC_BROWSER_TEST_F(JSEthereumProviderBrowserTest, EIP6369) {
   EXPECT_TRUE(
       content::EvalJs(primary_main_frame(), kEvalIsBraveWallet).ExtractBool());
 
-  std::string command = R"(
-    (async () => {
-      try {
-        let promise = new Promise((resolve) => {
-          const listener = (event) => {
-            window.removeEventListener("eip6963:announceProvider", listener);
-            let is_brave_wallet = event.detail.info.name === "Brave Wallet" &&
-                                  event.detail.provider.isBraveWallet === true;
-            resolve(is_brave_wallet);
-          }
-          window.addEventListener("eip6963:announceProvider", listener);
-          window.dispatchEvent(new Event("eip6963:requestProvider"));
-        })
-        return await promise;
-      } catch (e) {
-        return false;
-      }
-    })();)";
-
-  EXPECT_TRUE(content::EvalJs(primary_main_frame(), command).ExtractBool());
+  EXPECT_TRUE(
+      content::EvalJs(primary_main_frame(), kTestEIP6963).ExtractBool());
 
   EXPECT_EQ(browser()->tab_strip_model()->GetTabCount(), 1);
 }
 
 #if BUILDFLAG(ENABLE_EXTENSIONS)
 
 IN_PROC_BROWSER_TEST_F(JSEthereumProviderBrowserTest,
-                       EIP6369_MetaMaskAttached) {
+                       EIP6369_WithoutEthereumInstalled) {
   auto* keyring_service =
       brave_wallet::KeyringServiceFactory::GetServiceForContext(
           browser()->profile());
   keyring_service->CreateWallet("password", base::DoNothing());
 
-  scoped_refptr<const extensions::Extension> extension(
-      extensions::ExtensionBuilder("MetaMask")
-          .SetID(metamask_extension_id)
-          .Build());
-  extensions::ExtensionSystem::Get(browser()->profile())
-      ->extension_service()
-      ->AddExtension(extension.get());
-
   brave_wallet::SetDefaultEthereumWallet(
       browser()->profile()->GetPrefs(),
-      brave_wallet::mojom::DefaultWallet::BraveWallet);
+      brave_wallet::mojom::DefaultWallet::None);
```

Comment:
This is not right, with None there should be nothing injected, `window.braveEthereum` nor `window.ethereum`.
And EIP 6963 won't work in that scenario. What we need to test is install MetaMask and set the option to prefer extension and let MM overwrite `window.ethereum` to see if EIP 6963 works. 
Note that this is impossible to achieve before we add `window.braveEthereum`